frontend/packages/topology/integration-tests/support/step-definitions/topology/list-view.ts

```diff
@@ -0,0 +1,52 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { navigateTo } from '@console/dev-console/integration-tests/support/pages/app';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants/global';
+import { topologyPO } from '../../page-objects/topology-po';
+
+When('user clicks on List view button', () => {
+  navigateTo(devNavigationMenu.Topology);
+  if (cy.get(topologyPO.graph.emptyGraph)) {
+    cy.get(topologyPO.switcher).click();
+  } else {
+    cy.log('You are already on List View');
+  }
+});
+
+Then('user will see workloads are segregated by applications groupings', () => {
+  cy.get(topologyPO.graph.applicationGroupingTitle).should('be.visible');
+});
+
+When('user clicks the filter by resource on top', () => {
+  cy.get(topologyPO.graph.filterByResource.filterByResourceDropDown)
+    .should('contain', 'Filter by resource')
+    .click();
+});
+
+Then('user will see Deployment and Deployment-Config options with 1 associated with it', () => {
+  cy.get(topologyPO.graph.filterByResource.deploymentResource).should('contain', 'Deployment(1)');
+  cy.get(topologyPO.graph.filterByResource.deploymentConfigResource).should(
+    'contain',
+    'DeploymentConfig(1)',
+  );
+});
+
+Then('user clicks on Deployment checkbox to see only the deployment type workload', () => {
+  cy.get(topologyPO.list.resourceTitle)
+    .should('contain', 'Deployment')
+    .should('be.visible');
+  cy.get(topologyPO.list.resourceTitle)
+    .should('contain', 'DeploymentConfig')
+    .should('not.exist');
+});
+
+Then(
+  'user clicks on DeploymentConfig checkbox to see only the deploymentconfig type workload',
+  () => {
+    cy.get(topologyPO.list.resourceTitle)
+      .should('contain', 'DeploymentConfig')
+      .should('be.visible');
+    cy.get(topologyPO.list.resourceTitle)
+      .should('contain', 'Deployment')
+      .should('not.exist');
+  },
+);
```

Comment:
Are these tests executed as part of this PR??

Reply:
No. It was but we will use it later.

Reply:
All of these tests don't run, so either update gherkin or remove from the PR